packages/svelte/package.json

```diff
@@ -2,45 +2,48 @@
   "name": "@tolgee/svelte",
   "version": "4.8.0",
   "scripts": {
-    "build": "svelte-kit package",
+    "dev": "vite dev",
+    "build": "svelte-kit sync && svelte-package",
+    "develop": "sleep 5 && nodemon --exec svelte-package",
     "clean": "rm -rf build build-e2e dist dist-e2e lib .svelte-kit",
-    "preview": "svelte-kit preview",
-    "develop": "sleep 5 && nodemon --exec svelte-kit package",
```

Comment:
we use the develop script for development. Don't remove this.

Reply:
It moved, but hasn't been removed.